src/memory_dump/counters.c

```diff
@@ -18,7 +18,9 @@
  *****************************************************************************/
 #include <string.h>
 #include <err.h>
-#include "dump.h"
+
+#include "../ccan/ccan/htable/htable.h"
```

Comment:
Is `counters.c` still needed. Since the new version only captures one region at a time, we may not need a hash table to keep track of the current invocation, a simple counter suffices.

Can this file be removed ?